apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.ts

```diff
@@ -1056,8 +1056,8 @@ export class ProformaInvoiceComponent implements OnInit, OnDestroy, AfterViewIni
                         }
                     }
                     //  If last invoice copied then no need to add voucherAdjustments as pre-fill ref:- G0-5554
-                    if (this.isSalesInvoice) {
-                        if (results[1] && results[1].voucherAdjustments && results[1].voucherAdjustments.adjustments && results[1].voucherAdjustments.adjustments.length && !this.isLastInvoiceCopied) {
+                    if (this.isSalesInvoice || results[1] && results[1].voucherAdjustments) {
```

Comment:
add braces before results[1] otherwise condition will be skipped if voucherAdjustments won't be there.